.ci_support/linux_64_nodejs16.yaml

```diff
@@ -0,0 +1,25 @@
+c_compiler:
+- gcc
+c_compiler_version:
+- '9'
+cdt_name:
+- cos7
+channel_sources:
+- conda-forge
+channel_targets:
+- conda-forge main
+cxx_compiler:
+- gxx
+cxx_compiler_version:
+- '9'
+docker_image:
+- quay.io/condaforge/linux-anvil-cos7-cuda:10.2
```

Comment:
This seems odd. Not that it should cause any issues,  but wouldn't expect this image to be used here